src/components/LastDetection.tsx

```diff
@@ -0,0 +1,29 @@
+import Button from '@mui/material/Button'
+
+interface dateProps {
+  date: Date
+}
+
+export const LastDetection = (props: dateProps): JSX.Element => {
+  const currentDate: Date = new Date()
+  const lastDate: Date = new Date(props.date)
+  lastDate.toUTCString()
+  currentDate.toUTCString()
+  const dateDiff = Math.abs(+lastDate - +currentDate)
+  const dayDiff = Math.ceil(dateDiff / (1000 * 60 * 60 * 24))
+  const hoursDiff = Math.abs(+lastDate - +currentDate) / 36e5
+  return (
+    <>
+      <Button variant="contained">
+        {dateDiff < 8.64e7
```

Comment:
extract this into a readable variable (millisInADay) and use here and on L13 instead of having two different representations of the value.